src/plugins/Migrations/controllers/commands/migrateGuilds.js

```diff
@@ -0,0 +1,129 @@
+const db = require("quick.db");
+const dbh = require("../../../.common/structures/DataHandling/DatabaseHandler.js");
+const { Guild, Channel } = require("./../../../../../api/models");
+const migrationsHandler = require("./../../structures/MigrationHandler.js");
+
+exports.help = {
+  name: "migrateGuilds",
+  description: "Migrates all the data for the guilds",
+  usage: "migrateGuilds",
+  example: "migrateGuilds",
+};
+
+exports.conf = {
+  aliases: [],
+  cooldown: 5,
+  permissions: ["ADMINISTRATOR"],
+};
+
+exports.run = async (client, message, args) => {
+  const guilds = client.guilds.cache;
+  for (const guild of guilds) {
+    const guildId = message.guild.id;
+    const guildInDb = await Guild.findOne({ where: { id: guildId } });
+
+    if (guildInDb !== null) {
+      message.channel.send(`${guild[1].name} already has been created`);
+      continue; //Skips to the next guild;
+    }
+
+    /*For things that can be copy-pasted, with no modifications being made to the data,
+        we will use the system below
+      */
+    let keys = [
+      { oldKey: `prefix.${guildId}`, newKey: "prefix" },
+      { oldKey: `${guildId}.newUserRoles`, newKey: "defaultMemberRoles" },
+      { oldKey: `moderation-server.${guildId}`, newKey: "moderationServer" },
+    ];
+
+    const guildObject = migrationsHandler.copyData(keys);
+
+    guildObject.id = guildId;
+
+    const opRoles = [];
+    const modRole = db.get(`moderator.${guildId}`);
+    const adminRole = db.get(`admin.${guildId}`);
+    if (modRole !== null) opRoles.push(modRole);
+    if (adminRole !== null) opRoles.push(adminRole);
+    if (opRoles.length > 0) guildObject.opRoles = opRoles;
+
+    const levels = [];
+    const levelNums = [
```

Comment:
Are these values also defined elsewhere?

Reply:
They are defined in the old data base under the key levels.${the id of the guild}.${the level you are trying to access}

Reply:
Oh I thought it's always appended at the end.